examples/wearable/UIComponents/contents/navigationelements/fastscroll.js

```diff
@@ -0,0 +1,21 @@
+(function (tau) {
+	var page = document.getElementById("fast-scroll-page"),
+		sh = null;
```

Comment:
How about scrollHandler instead of sh abbreviation?

Reply:
Done